Phish List

Zachinquarantine opened the pull request:
`www.roblox.com.es`
Full URL: hxxps://www.roblox.com.es/users/916562743/profile
Roblox phish with almost identical URL
`newappmail.discoverynevers.com` and `ee-updatebillingform.com`
Full URLs: hxxps://newappmail.discoverynevers.com/EEBILL4JXHZB4J | hxxps://ee-updatebillingform.com/account/index?ac=ee
More info: https://media.discordapp.net/attachments/857078630524649492/864161119979175956/unknown.png?width=947&height=473
`app-uniswap.org-vote-eth-swap.site` and `app-uniswap.org-claim-eth-liquidty.site`
Identical crypto phishes, full URLs: hxxps://app-uniswap.org-vote-eth-swap.site/v2/ and hxxps://app-uniswap.org-claim-eth-liquidty.site/v2/
`tracktt.deliveryoffer.top`
IPS phish, full URL: hxxps://tracktt.deliveryoffer.top/pt/ips/?p=200&cep=X9BH9gKIBcspdlDcnd7KokJ6RiWP2gR6Yu9TpgF6lV6Doi849_z7sZZPytzTyCvRL55Vih8bToYzdxFbA-AGHikPoJQDyo62Csr7oHuTzIEeyV22UiD5DapTjhti3Fb6eEiyo9OUo-iT5a-TouEvB5h2CY3uBYvgKNwCUMjGUDpl6jOeCFftHyV5pGFFZmECDQtQ0bp32qUnnkJ_sjxWjFvy-JfO4n1jq751e1uAM7HVVvFXgUlxY8ODT0DUxCZ9keN3JGrK8nHE4Z6YgtZs1M2SYEv71oNxcBm0Fx2UXWRP7-1Or2-sHaSyJ5DNqg5U4oUVcYZZxWcRfMyC9D1upFJo_z21up8qYzS_A8pdnCSByDeXYLKXrxTOrlxZP0Mv6Y9z5Mpnw9qy7d5xPqE7Ofm0zlgp7pZhd7R76Uzua2sd6QfwVPT8HVak4MZ_RRICb1UviiCObgl1Ahno13UB1xp4a6ReRp_Z7AjNsjA755b1WZGlwiGA_wValnakf3od&lptoken=164026c403f877852882&click_id=60unMhG&var2=&var3=Y60EADA5A4AA86&var4=&var5=10&var6=&var7=&var8=&var9=351938364198&var10=

Commit:
Phishes #1 for 2021-07-12

`www.roblox.com.es`
Full URL: hxxps://www.roblox.com.es/users/916562743/profile
Roblox phish with almost identical URL
`newappmail.discoverynevers.com` and `ee-updatebillingform.com`
Full URLs: hxxps://newappmail.discoverynevers.com/EEBILL4JXHZB4J | hxxps://ee-updatebillingform.com/account/index?ac=ee
More info: https://media.discordapp.net/attachments/857078630524649492/864161119979175956/unknown.png?width=947&height=473
`app-uniswap.org-vote-eth-swap.site` and `app-uniswap.org-claim-eth-liquidty.site`
Identical crypto phishes, full URLs: hxxps://app-uniswap.org-vote-eth-swap.site/v2/ and hxxps://app-uniswap.org-claim-eth-liquidty.site/v2/
`tracktt.deliveryoffer.top`
IPS phish, full URL: hxxps://tracktt.deliveryoffer.top/pt/ips/?p=200&cep=X9BH9gKIBcspdlDcnd7KokJ6RiWP2gR6Yu9TpgF6lV6Doi849_z7sZZPytzTyCvRL55Vih8bToYzdxFbA-AGHikPoJQDyo62Csr7oHuTzIEeyV22UiD5DapTjhti3Fb6eEiyo9OUo-iT5a-TouEvB5h2CY3uBYvgKNwCUMjGUDpl6jOeCFftHyV5pGFFZmECDQtQ0bp32qUnnkJ_sjxWjFvy-JfO4n1jq751e1uAM7HVVvFXgUlxY8ODT0DUxCZ9keN3JGrK8nHE4Z6YgtZs1M2SYEv71oNxcBm0Fx2UXWRP7-1Or2-sHaSyJ5DNqg5U4oUVcYZZxWcRfMyC9D1upFJo_z21up8qYzS_A8pdnCSByDeXYLKXrxTOrlxZP0Mv6Y9z5Mpnw9qy7d5xPqE7Ofm0zlgp7pZhd7R76Uzua2sd6QfwVPT8HVak4MZ_RRICb1UviiCObgl1Ahno13UB1xp4a6ReRp_Z7AjNsjA755b1WZGlwiGA_wValnakf3od&lptoken=164026c403f877852882&click_id=60unMhG&var2=&var3=Y60EADA5A4AA86&var4=&var5=10&var6=&var7=&var8=&var9=351938364198&var10=

Commit:
mail-e9eredelivery.com

Commit:
add hxxps://www.kunestra.hr/.uk/

Commit:
IPS phish

Full URL: hxxps://trackpp.searchmyitem.xyz/pt/ips/?p=200&cep=kR4f9jPSpH0bQxVULPZxptV0as3f6BqUBGuIyNZwwKHNLblb3LJoYLAR2E3q9fhuvBzQ9Hw2DtQJSaVCUe8cQA3D0DRL6pFwqCvoN3cAts_8YNhWWpJiR665gAql1OUjn2BABt5v2g73r1KRrq3PO4Kchtfcq-oUVwrOizVmV3ZiiTcejzVXYmUISh8QhfV37WKz-0TqBtBy7PaK4sGMOW2FAzedHsnNolbB2HMqoQM33iNn1Sg3VEhTj9DH-TrhJGnKzJ8rRURz4a-hpVlmp0TCqQVIAEvMfhFktEaD464QzwCHvwj7nhOi65SnPw8pGYlyrHYVRzVdC_1wcNv9672e04-yL33KM5Zp2Kf7oG5VWsOtT5bYhOKtdclVxDDnVidS8UhgN8hnx9AJ3pC1GF5e8skLU_QIrL_D4CsUaiXLc9OZywE-TF6NW5-13bzf-UBwpEa7RW7CYkWZD-cOMR3uV3zuGa1ajQHNK-6UL04&lptoken=168826a9202c501732b2&click_id=KVQzMDz&var2=&var3=P60ED659B79001&var4=&var5=31&var6=&var7=&var8=&var9=351934431720&var10=

Commit:
steam phish

Commit:
at&t phishes

Commit:
league of legends phish

Zachinquarantine commented:
can someone check this pr please? @spirillen @mitchellkrogza

spirillen commented on add-link:
```diff
@@ -38,3 +38,4 @@ https://www.mdscbenin.org/wp-includes/theme-compat/dar/newsant1/login.php
 https://nort2on.com/setup/
 http://eternalstarculture.com/intern/home.php
 https://tinyurl.com/4cbf4wcp
+https://www.kunestra.hr/.uk/
```
Confirmed, but you should be aware it is also working on all four versions of 

```js
http(s)?://(www.)?kunestra.hr/.uk/
```

![image](https://user-images.githubusercontent.com/44526987/126137228-6db83872-6a6a-4b0b-86f5-ffbcb33a847d.png)

spirillen commented on add-domain:
```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+www.roblox.com.es
```
Can't confirm

![image](https://user-images.githubusercontent.com/44526987/126138178-7747552a-b96f-4b9c-af2c-ac15c940c263.png)

spirillen commented on add-domain:
```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+www.roblox.com.es
+newappmail.discoverynevers.com
```
Seems to have been solved

![image](https://user-images.githubusercontent.com/44526987/126139350-71112c3f-0043-4177-af56-dcf195ab7230.png)

spirillen commented on add-domain:
```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
 totaltrkonups2298.fitritomoastramencostea.com
+www.roblox.com.es
+newappmail.discoverynevers.com
+app-uniswap.org-vote-eth-swap.site
```
Seems dead to me from both manual lookup and @pyfunceble

![image](https://user-images.githubusercontent.com/44526987/126140881-7db9491e-8d72-48e2-a1f7-6fd0e9b227cd.png)

spirillen commented on add-domain:
```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
+www.roblox.com.es
+newappmail.discoverynevers.com
+app-uniswap.org-vote-eth-swap.site
+app-uniswap.org-claim-eth-liquidty.site
```
Seems dead to me from both manual lookup and @pyfunceble

![image](https://user-images.githubusercontent.com/44526987/126141021-3efc7be7-28b5-4fb7-aa77-57d0428e1146.png)

spirillen reviewed (changes requested):
I have now spend 45 trying to verify this bulk load of domains, and the main part of those I've tested seems dead.

Please take a second look at your submission

Commit:
remove dead domains

spirillen commented on add-domain:
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
```
drill -T deliveryoffer.top        
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      m.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
top.    172800  IN      NS      a.zdnscloud.com.
top.    172800  IN      NS      b.zdnscloud.com.
top.    172800  IN      NS      c.zdnscloud.com.
top.    172800  IN      NS      d.zdnscloud.com.
top.    172800  IN      NS      f.zdnscloud.com.
top.    172800  IN      NS      g.zdnscloud.com.
top.    172800  IN      NS      i.zdnscloud.com.
top.    172800  IN      NS      j.zdnscloud.com.
top.    3600    IN      SOA     a.zdnscloud.com. td_dns_gtld.knet.cn. 1626793776 600 200 2491200 3600

spirillen commented on add-domain:
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
+mail-e9eredelivery.com
```
drill -T mail-e9eredelivery.com
.       518400  IN      NS      m.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
com.    172800  IN      NS      k.gtld-servers.net.
com.    172800  IN      NS      g.gtld-servers.net.
com.    172800  IN      NS      e.gtld-servers.net.
com.    172800  IN      NS      h.gtld-servers.net.
com.    172800  IN      NS      l.gtld-servers.net.
com.    172800  IN      NS      i.gtld-servers.net.
com.    172800  IN      NS      d.gtld-servers.net.
com.    172800  IN      NS      a.gtld-servers.net.
com.    172800  IN      NS      m.gtld-servers.net.
com.    172800  IN      NS      j.gtld-servers.net.
com.    172800  IN      NS      c.gtld-servers.net.
com.    172800  IN      NS      b.gtld-servers.net.
com.    172800  IN      NS      f.gtld-servers.net.
com.    900     IN      SOA     a.gtld-servers.net. nstld.verisign-grs.com. 1626793872 1800 900 604800 86400

spirillen commented on add-domain:
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
+mail-e9eredelivery.com
+trackpp.searchmyitem.xyz
```
drill -T searchmyitem.xyz
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
.       518400  IN      NS      m.root-servers.net.
xyz.    172800  IN      NS      x.nic.xyz.
xyz.    172800  IN      NS      generationxyz.nic.xyz.
xyz.    172800  IN      NS      z.nic.xyz.
xyz.    172800  IN      NS      y.nic.xyz.
xyz.    3600    IN      SOA     ns0.centralnic.net. hostmaster.centralnic.net. 3000620592 900 1800 6048000 3600

spirillen commented on add-domain:
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
+tracktt.deliveryoffer.top
+mail-e9eredelivery.com
+trackpp.searchmyitem.xyz
+steamcomnnulty.com
```
Looks alive :+1: 

![image](https://user-images.githubusercontent.com/44526987/126349434-d7d4179f-5312-40e8-9ed5-b931e16826e5.png)

spirillen commented on add-domain:
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
+mail-e9eredelivery.com
+trackpp.searchmyitem.xyz
+steamcomnnulty.com
+yahoofbuyv.weebly.com
```
Taken down, See also Special Rules in Pyfunceble 

https://github.com/funilrys/PyFunceble/blob/89f2b69c4f02657124d3a05d6b73e43ca5a46c06/PyFunceble/checker/availability/extra_rules.py#L99-L101

from commit https://github.com/funilrys/PyFunceble/commit/89f2b69c4f02657124d3a05d6b73e43ca5a46c06

spirillen commented on add-domain:
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
+steamcomnnulty.com
+yahoofbuyv.weebly.com
+attonlineservicexx.weebly.com
+julyemailclosurer.weebly.com
```
`pyfunceble -d julyemailclosurer.weebly.com attonlineservicexx.weebly.com yahoofbuyv.weebly.com`

You are using the Beta version of PyFunceble 4.0.0!
Please take the time to communicate with us when you notice
something unusual.




Subject                                                                                              Status      Source    
---------------------------------------------------------------------------------------------------- ----------- ----------
`attonlineservicexx.weebly.com`                                                                        INACTIVE    SPECIAL   
`julyemailclosurer.weebly.com`                                                                         INACTIVE    SPECIAL   
`yahoofbuyv.weebly.com `                                                                               INACTIVE    SPECIAL

spirillen commented on add-domain:
```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
+yahoofbuyv.weebly.com
+attonlineservicexx.weebly.com
+julyemailclosurer.weebly.com
+support-1oakyf-league-of-legends.000webhostapp.com
```
Active:

![image](https://user-images.githubusercontent.com/44526987/126351255-cd920897-b576-458c-aaeb-13b047f916f9.png)


You can test this with the special rules for @PyFunceble: 

https://github.com/funilrys/PyFunceble/blob/89f2b69c4f02657124d3a05d6b73e43ca5a46c06/PyFunceble/checker/availability/extra_rules.py#L84-L86

spirillen reviewed (changes requested):
I asked you to take a second look at these domains, there are still a lot of :skull: once.

This is exactly one of many reasons I do one domain = one issue at https://mypdns.org/my-privacy-dns to make sure you can get stuff merged and distributed A.S.A.P

spirillen commented:
@Zachinquarantine I can recommend you to use @PyFunceble to test submissions before adding them. PyFunceble is written for Linux, but it have with some success been used on windows within PowerShell and Cygwin by @DandelionSprout

PyFunceble at GH: https://github.com/funilrys/PyFunceble
PyFunceble at MypDNS (mirror): https://mypdns.org/pyfunceble/PyFunceble
PyFunceble at pypi: https://pypi.org/project/PyFunceble-dev/
PyFunceble at Read the docs: https://pyfunceble.readthedocs.io/en/dev/

Commit:
remove more dead domains

Commit:
removes kunestra.hr/.uk (phishing page was removed)

Zachinquarantine commented:
@spirillen

Commit:
add dlscord.shop

full url: hxxps://dlscord.shop/steam/giveaway-nitro
steam phishing scam

Commit:
add http-web-roblox.com

Full URL: hxxp://http-web-roblox.com/users/657483709/profile
https://www.virustotal.com/gui/url/d17a4fafd676ca5c8061f96d25581f72163a32933d1e774f6b79fbfe5d075902/detection
Roblox phishing

spirillen commented on add-domain:
```diff
@@ -522,3 +522,10 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
```
Note for others.

This domain seems inactive at the writing moment; However the domain is still active, which means the DNS/web-server etc can be activated within milliseconds or only be response to specific regions / IP addresses, Hence leaving this record as ACTIVE

spirillen commented on add-domain:
```diff
@@ -522,3 +522,10 @@ pdfkinghub.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
+mail-e9eredelivery.com
```
Note for others.

This domain seems inactive at the writing moment; However the domain is still active, which means the DNS/web-server etc can be activated within milliseconds or only be response to specific regions / IP addresses, Hence leaving this record as ACTIVE

spirillen reviewed (approved):
Approved with comments for others to read
